client/allocator/Cargo.toml

```diff
@@ -17,14 +17,15 @@ targets = ["x86_64-unknown-linux-gnu"]
 sp-std = { version = "3.0.0", path = "../../primitives/std", default-features = false }
 sp-core = { version = "3.0.0", path = "../../primitives/core", default-features = false }
 sp-wasm-interface = { version = "3.0.0", path = "../../primitives/wasm-interface", default-features = false }
-log = { version = "0.4.11", optional = true }
+log = { version = "0.4.11" }
 thiserror = { version = "1.0.21" }
 
 [features]
 default = [ "std" ]
+track-heap = []
```

Comment:
Why do we want to introduce this feature? Is this only about performance? If so, I do not think that a couple of branches and ALU ops will make any difference.

Reply:
yeah. Since the feature is pretty simple and need not be propagated to many places, I think it is okay to keep it.

Reply:
I'm also for removing it. The problem is that it would be activated anyway everywhere. (thanks cargo). So, we can just enable it all the time directly.

Reply:
I see this as introducing complexity. I do agree that it is not a lot, but it is there. On the opposite side, there is literally no benefit (I do not count theoretical speed up which is most likely is not measurable compared to noise).

Reply:
fair enough 🎩 .